pulp_rpm/app/tasks/publishing.py

```diff
@@ -365,10 +365,10 @@ def create_repomd_xml(
         cwd = os.path.join(cwd, sub_folder)
         repodata_path = os.path.join(sub_folder, repodata_path)
 
-    if package_checksum_type and package_checksum_type not in settings.ALLOWED_CONTENT_CHECKSUMS:
+    if package_checksum_type not in settings.ALLOWED_CONTENT_CHECKSUMS:
```

Comment:
I don't think ```package_checksum_type``` is guaranteed to not be ```None```, is it?

Reply:
I believe https://github.com/pulp/pulp_rpm/blob/master/pulp_rpm/app/tasks/publishing.py#L297 guaranteed it.

Reply:
Ok